tajo-sql-parser/src/main/antlr4/org/apache/tajo/parser/sql/SQLParser.g4

```diff
@@ -106,9 +106,9 @@ if_exists
   ;
 
 create_table_statement
-  : CREATE EXTERNAL TABLE (if_not_exists)? table_name table_elements (TABLESPACE spacename=identifier)? USING storage_type=identifier
+  : CREATE EXTERNAL TABLE (if_not_exists)? table_name (table_elements)? (TABLESPACE spacename=identifier)? USING storage_type=identifier
```

Comment:
This grammar allows to create table without parenthesis ``(..)``. It looks good to me.

I also have another suggestion. How do you think about a special notation with parenthesis like ``CREATE EXTERNAL TABLE (*) USING JSON``. I used asterisk ``*`` as a meaning of becoming anything and the infinite number of fields, which can be zero to up to be infinite.